roles/k3s/master/tasks/main.yml

```diff
@@ -160,6 +160,13 @@
       --kubeconfig ~{{ ansible_user }}/.kube/config
   changed_when: true
 
+- name: Fetch cluster config file from first master to push to agent nodes - allowing agent nodes to run k3s kubectl xxx (patching, convenience)
```

Comment:
This change was also included in https://github.com/techno-tim/k3s-ansible/pull/101 which we aren't going to merge